components/eamxx/src/share/io/scorpio_output.hpp

```diff
@@ -178,6 +178,7 @@ class AtmosphereOutput
   // sim_field_manager points to the simulation field manager
   // when remapping horizontally these two field managers may be different.
   std::shared_ptr<const fm_type>      m_io_field_mgr;
+  std::shared_ptr<const fm_type>      m_int_field_mgr;
```

Comment:
I'm guessing `int` stands for `intermediate`, but it's awfully reminding of the `int` data type. Perhaps we can spell "intermediate" out? Or something more descriptive, like `m_field_mgr_after_vinterp`?